### PR TITLE
New PR Template

### PR DESCRIPTION
**Pull request by ShivamTyagi12345**
This pull request proposes to add a new template for pull requests to the repository. The new template provides a clear and concise format for contributors to follow when submitting changes to the codebase. The template includes sections for describing the problem, solution, changes made, instructions for testing, relevant screenshots, notes, and a checklist.

The goal of this pull request is to ensure that all pull requests have a consistent format and provide enough information for reviewers to easily understand the changes and assess their impact on the codebase. This will help streamline the review process, reduce the time spent on reviewing, and increase the overall efficiency of the development workflow.

The new template has been designed to be flexible and can be customized to suit the specific needs of the repository. The proposed template is well-documented and provides a clear structure for contributors to follow, helping them to present their changes in a professional and organized manner.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+
+## Notes
+
+[Any additional notes or information that you would like to share with the reviewers.]
+
+## Checklist
+
+- [ ] Code follows the code style and formatting guidelines
+- [ ] Changes have been thoroughly tested
+- [ ] Documentation has been updated, if necessary
```
Comment (mlabouardy) [inline PULL_REQUEST_TEMPLATE.md:29]: can you link it to https://github.com/tailwarden/docs.komiser.io repo

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
+
+[Include screenshots, if relevant, to help reviewers understand the changes you made.]
+
+## Notes
+
+[Any additional notes or information that you would like to share with the reviewers.]
+
+## Checklist
+
+- [ ] Code follows the code style and formatting guidelines
```
Comment (mlabouardy) [inline PULL_REQUEST_TEMPLATE.md:27]: can you link it to https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md

**Review by mlabouardy — CHANGES_REQUESTED** [review]
Thanks @ShivamTyagi12345, request small changes to improve the template

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ShivamTyagi12345) [reply PULL_REQUEST_TEMPLATE.md]: done

**Review by mlabouardy — APPROVED** [review]
Thanks for the changes!

**Pull request merged**